pkg/syncer/spec/mutators/deployment.go

```diff
@@ -44,7 +46,9 @@ type DeploymentMutator struct {
 	syncTargetClusterName logicalcluster.Name
 	syncTargetUID         types.UID
 	syncTargetName        string
+	syncTargetKey         string
```

Comment:
key in which sense?

Reply:
The one used in labels, and annotations. I'll comment.

Reply:
I finally simplified the code so that this field is not required anymore.